examples/LGACSend.ino

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017 xpokor22
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+#include <ir_LG.h>
+
+IRsend irsend(14);  // An IR LED is controlled by GPIO pin 14 (D5)
+
+// 0 : TOWER
+// 1 : WALL
+const unsigned int AC_TYPE  = 1;
+
+// 0 : cooling
+// 1 : heating
+unsigned int AC_HEAT = 1;
```

Comment:
lowercase for all variables please. https://google.github.io/styleguide/cppguide.html#Variable_Names
We follow the Google c++ style guide for this project/library.

This comment applies to all the other instances/variables below. #define(s) are fine in UPPER_CASE.